src/main/java/reposense/parser/CsvParser.java

```diff
@@ -207,4 +226,9 @@ private String getRowContentAsRawString(final CSVRecord record) {
      * {@code record} and add created objects into {@code results}.
      */
     protected abstract void processLine(List<T> results, final CSVRecord record) throws ParseException;
+
+    /**
+     * Gets the number of columns expected in the header.
```

Comment:
Better starting word to use  here (same for those below) is `Returns the number of expected cols ...`

Reply:
I removed the `getHeader` method entirely, since we are passing the expected header size to the constructor of `CsvParser`